cmd/build.go

```diff
@@ -51,6 +59,8 @@ func init() {
 	buildCmd.Flags().StringVarP(&DockerImage, "image", "i", "", "Image name and tag of resulting image")
 	buildCmd.Flags().StringVarP(&DockerContext, "context", "c", ".", "Path to a directory containing a Dockerfile, it is build context that is sent to the Docker daemon")
 	buildCmd.Flags().BoolVarP(&PushImage, "push", "p", false, "Add this flag if you want to push the image")
+	buildCmd.Flags().BoolVarP(&s2iBuild, "s2i", "", false, "If this is enabled then BuildConfig and Imagestream is created")
```

Comment:
This should say that it will use S2I, If BuildConfing is created it doesn't mean that S2I will be used for building

Reply:
please update description. It should mention that it will create BuildConfig with SourceStragegy

Reply:
done